packages/replayio/src/utils/spawnProcess.ts

```diff
@@ -29,30 +34,28 @@ export function spawnProcess(
     // github.com/replayio/replay-cli/pull/344#discussion_r1553258356
     spawned.unref();
   } else {
+    const stderr = collectData(spawned.stderr);
+
     spawned.on("error", error => {
       deferred.rejectIfPending(error);
-
-      if (onError) {
-        onError(error);
-      }
     });
 
     spawned.on("spawn", () => {
-      if (onSpawn) {
-        onSpawn();
-      }
+      onSpawn?.();
```

Comment:
Totally subjective and fine to leave this but oof I hate this syntax 😆

Reply:
This guy is sad to hear that 😢 
<img width="38" alt="?.(); put vertically, question mark at the top, semicolon at the bottom. It resembles a guy with an open mouth" src="https://github.com/replayio/replay-cli/assets/9800850/f515b92b-183c-42e8-85cd-3fe8ab3105d6">

Reply:
༼ ༎ຶ ෴ ༎ຶ༽